### ISSUE TITLE
 test_gan_3  error

### ISSUE THREAD

**Issue by lyzhangjm** [layumi/2016_GAN_Matlab#4]
Thanks for your response.  I can run the train_gan_3.m,  however, when I run the  test_gan_3, there is a error:
>> test_gan_3
错误使用 vl_nnconv
FILTERS are larger than the DATA (including padding).
出错 dagnn.Conv/forward (line 11)
      outputs{1} = vl_nnconv(...
出错 dagnn.Layer/forwardAdvanced (line 85)
      outputs = obj.forward(inputs, {net.params(par).value}) ;
出错 dagnn.DagNN/eval (line 88)
  obj.layers(l).block.forwardAdvanced(obj.layers(l)) ;
出错 test_gan_3 (line 11)
net1.eval({'data_rand',(im)});

 I don't modify any parameters.   Can you run the test_gan_3 ?
